features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -4080,4 +4080,10 @@
     </CacheInvalidator>
     {% endif %}
 
+    <ConfigSwitching>
```

Comment:
As a practice, we add these configs to identity.xml file as well. Shall we add this to identity.xml file?

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/5744/commits/8132e60b7a407cfa161ceef409dc534a74e7dae2